### DISCUSSION TITLE
Mender update failure

### DISCUSSION THREAD

**Discussion by tirankas** [OE4T/tegra-demo-distro#77] [General]
Hello,
I am trying to install an update file and see a failure.
My setup is based on the dunfell 32.5.0.
Using Jetson NX Xavier, emmc based, with the following command shows:

strace -ff -onvup -s256 /usr/sbin/nv_update_engine --install no-reboot
Nvidia A/B-Redundancy Update tool Version 2.0
current slot 0
SM: S11
Set slot 1 as unbootable and start updating.
Start running: /usr/sbin/nv_bootloader_payload_updater --no-dependent-partition /opt/ota_package/bl_update_payload
Got update payload: /opt/ota_package/bl_update_payload
Tegra User Block Device: /dev/disk/by-partlabel
Tegra Boot Block Device: /dev/mtdblock0
HEADER: MAGIC NVIDIA__BLOB__V2
HEX_VALUE 131072
BLOB_SIZE 48002916
HEADER_SIZE 48
NUMBER_OF_ELEMENTS 29
HEADER_TYPE 0
UNCOMP_SIZE 48002916
MB1_RATCHET_LV 0
MTS_RATCHET_LV 0
ROLLBACK_FUSE_LV 0
config COMPATIBLE_SPEC not found in /etc/nv_boot_control.conf
Device TN Spec: 3668-200-0001-G.0-1-2-jetson-xavier-nx-devkit-emmc-juganu-mmcblk0p1
Can't find matching TN Spec in OTA Blob!
OTA Blob update failed. Status: 3
/usr/sbin/nv_bootloader_payload_updater --no-dependent-partition failed.
Slot 1 update failed
Error: installing bootloader updates failed: 768

What am I missing?

**Comment by madisongh** [comment]
You need to shorten your custom machine name to 31 characters or less.  See [this issue](https://github.com/OE4T/meta-tegra/issues/638).

**Reply by tirankas** [reply]
Thanks!
I will post back after the change.

**Reply by tirankas** [reply]
Thanks Matt.
It was indeed the issue, looks ok now...
